--- geolinks-0.2.1.bkp/setup.py	2023-02-26 13:26:23.000000000 -0300
+++ geolinks-0.2.1/setup.py	2023-02-27 09:25:19.325465532 -0300
@@ -32,7 +32,7 @@
 import re
 import sys
 
-from geolinks import __version__ as version
+# from geolinks import __version__ as version
 from setuptools import Command, find_packages, setup
 
 
@@ -73,7 +73,7 @@
 
 setup(
     name='geolinks',
-    version=version,
+    version="0.2.1",
     description='Utilities to deal with geospatial links',
     long_description=LONG_DESCRIPTION,
     long_description_content_type='text/markdown',
